include/aspect/simulator.h

```diff
@@ -484,16 +484,16 @@ namespace aspect
       /**
        * Compute the factor by which we scale the second of
        * the Stokes equations (the "pressure scaling factor").
-       * We do this for the current time step by taking some kind
-       * of average of the viscosities we find on the cells in this domain.
+       * We do this for the current time step by taking the average
+       * of the logarithm of the viscosities we find on the cells in this domain.
```

Comment:
That's not strictly true -- but maybe call it "logarithmic average" or something similar.